libsolidity/interface/CompilerStack.h

```diff
@@ -272,8 +273,10 @@ class CompilerStack: boost::noncopyable
 	StringMap loadMissingSources(SourceUnit const& _ast, std::string const& _path);
 	std::string applyRemapping(std::string const& _path, std::string const& _context);
 	void resolveImports();
```

Comment:
After this change, only `resolveImports` and `applyRemapping` are the undocumented functions.